background/setBackground.js

```diff
@@ -1,5 +1,38 @@
-const setBackground = (weatherCode) => {
-	const background = document.getElementById("background");
-	background.style.backgroundImage = `url(${backgroundsMapping[weatherCode]})`;
-	background.style.backgroundColor = '#efefef';
+const background = document.getElementById("background");
+
+const toggleBackground = (weatherCode) => {
+    if (weatherCode) {
+        setWeatherBackground(weatherCode);
+    }  else {
+        setInitialBackground();
+    }
+}
+
+const setWeatherBackground = (weatherCode) => {
+    const initialBackground = document.getElementById('initial-background');
+    if (document.body.contains(initialBackground)) {
```

Comment:
Why are you checking twice if element is in DOM. document.getElementById will be undefined if it's not in DOM. So instead, you can do:
```
if (initialBackground)
```

Reply:
Right :+1: